### ISSUE TITLE
WebAudio support

### ISSUE THREAD

**Issue by lexesv** [weolar/miniblink49#250]
Hello.

Build 2019.3.12

![Screenshot_2019-03-14_15-41-42](https://user-images.githubusercontent.com/15149258/54361463-b98f4800-466f-11e9-80fd-b219cf5fdb7f.png)

**Comment by weolar** [comment]
WebAudio has been removed...

**Comment by lexesv** [comment]
If I purchase the 个人版 , WebAudio & WebAsembly will be supported?

**Comment by weolar** [comment]
No...WebAudio is  heavy depends on chromium , it is hard to restore.
but WebAsembly is just depends on v8. I can updata newest v8 to support.
